pom.xml

```diff
@@ -27,6 +27,18 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-core</artifactId>
+            <version>1.2.9</version>
```

Comment:
It is a good practice to extract the dependency version into separate variable

Reply:
fix